Doc/library/pyexpat.rst

```diff
@@ -196,6 +196,37 @@ XMLParser Objects
    :exc:`ExpatError` to be raised with the :attr:`code` attribute set to
    ``errors.codes[errors.XML_ERROR_CANT_CHANGE_FEATURE_ONCE_PARSING]``.
 
+.. method:: xmlparser.SetReparseDeferralEnabled(enabled)
+
+   .. warning::
+
+      Calling ``SetReparseDeferralEnabled(False)`` has security implications,
+      as detailed below; please make sure to understand these consequences
+      prior to using the ``SetReparseDeferralEnabled`` method.
+
+   Expat 2.6.0 introduced a security mechanism called "reparse deferral"
+   where instead of causing denial of service through quadratic runtime
+   from reparsing large tokens, reparsing of unfinished tokens is now delayed
+   by default until a sufficient amount of input is reached.
+   Due to this delay, registered handlers may — depending of the sizing of
+   input chunks pushed to Expat — no longer be called right after pushing new
+   input to the parser.  Where immediate feedback and taking over responsiblity
+   of protecting against denial of service from large tokens are both wanted,
+   calling ``SetReparseDeferralEnabled(False)`` disables reparse deferral
+   for the current Expat parser instance, temporarily or altogether.
+   Calling ``SetReparseDeferralEnabled(True)`` allows re-enabling reparse
+   deferral.
+
+   .. versionadded:: 3.13 backported to 3.8.19, 3.9.19, 3.10.14, 3.11.9, 3.12.3
```

Comment:
It should be just
```suggestion
   .. versionadded:: 3.12.3
```

Reply:
@serhiy-storchaka okay, done. I'll go create the other backports now (3.10 down to 3.8).

How about `main` after we're done backporting — should it say anything more than `3.13` there then, maybe `3.13 backported to 3.8 and after` or `3.8, 3.9, ..` or or? I found one similar case from the past:

```console
# git grep "backported to" | head -n1
Doc/library/decimal.rst:.. versionadded:: 3.9 backported to 3.7 and 3.8.
```
What do you think?

Reply:
This is only one example. There are few examples in `typing.rst` with several consequent `versionadded` directives with different versions, but I think that they are incorrect. In most cases a single `versionadded` directive is used. If the feature was backported to bugfix releases, it points to the latest of such releases. In this case it should be 3.12.3.

Reply:
@serhiy-storchaka okay, I'll accept that for the way forward.